src/Angor/Avalonia/AngorApp/UI/Sections/Founder/CreateProject/Profile/ProfileViewModel.cs

```diff
@@ -29,38 +30,72 @@ public ProfileViewModel(CreateProjectFlow.ProjectSeed projectSeed, UIServices ui
+            x => string.IsNullOrWhiteSpace(x) || IsValidImageUrl(x),
+          "Banner URL must link to a valid image file (jpg, jpeg, png, gif, webp, svg)").DisposeWith(disposable);
+
+        // AVATAR URL VALIDATIONS (ALWAYS - Optional but must be valid if provided)
+        this.ValidationRule(x => x.AvatarUri,
+            x => string.IsNullOrWhiteSpace(x) || (Uri.TryCreate(x, UriKind.Absolute, out var uri) && (uri.Scheme == Uri.UriSchemeHttp || uri.Scheme == Uri.UriSchemeHttps)),
+          "Please enter a valid URL (starting with http:// or https://)").DisposeWith(disposable);
+        this.ValidationRule(x => x.AvatarUri,
+         x => string.IsNullOrWhiteSpace(x) || IsValidImageUrl(x),
+              "Profile image URL must link to a valid image file (jpg, jpeg, png, gif, webp, svg)").DisposeWith(disposable);
+
+        // NIP-05 and Lightning Address validations (async)
         var isValidNip05Username = this.WhenAnyValue(model => model.Nip05Username)
-            .Throttle(TimeSpan.FromSeconds(1), RxApp.MainThreadScheduler)
-            .Select(s => Observable.FromAsync(() => s == null ? Task.FromResult(Result.Success()) : uiServices.Validations.CheckNip05Username(s, projectSeed.NostrPubKey)))
-            .Switch();
-        
+        .Throttle(TimeSpan.FromSeconds(1), RxApp.MainThreadScheduler)
+        .Select(s => Observable.FromAsync(() => s == null ? Task.FromResult(Result.Success()) : uiServices.Validations.CheckNip05Username(s, projectSeed.NostrPubKey)))
+        .Switch();
+
         var isValidLightningAddress = this.WhenAnyValue(model => model.LightningAddress)
-            .Throttle(TimeSpan.FromSeconds(1), RxApp.MainThreadScheduler)
-            .Select(s => Observable.FromAsync(() => s == null ? Task.FromResult(Result.Success()) : uiServices.Validations.CheckLightningAddress(s)))
-            .Switch();
-        
+        .Throttle(TimeSpan.FromSeconds(1), RxApp.MainThreadScheduler)
+        .Select(s => Observable.FromAsync(() => s == null ? Task.FromResult(Result.Success()) : uiServices.Validations.CheckLightningAddress(s)))
+        .Switch();
+
         this.ValidationRule(x => x.Nip05Username, isValidNip05Username, x => x.IsSuccess, error => error.Error).DisposeWith(disposable);
-        
         this.ValidationRule(x => x.LightningAddress, isValidLightningAddress, x => x.IsSuccess, error => error.Error).DisposeWith(disposable);
 
         Errors = new ErrorSummarizer(ValidationContext).DisposeWith(disposable).Errors;
     }
 
-    public ICollection<string> Errors { get; }
+    private static bool IsValidImageUrl(string url)
+    {
+        if (string.IsNullOrWhiteSpace(url))
+            return false;
 
+        var lowerUrl = url.ToLowerInvariant();
+        return lowerUrl.EndsWith(".jpg") ||
+          lowerUrl.EndsWith(".jpeg") ||
+          lowerUrl.EndsWith(".png") ||
+          lowerUrl.EndsWith(".gif") ||
+          lowerUrl.EndsWith(".webp") ||
+          lowerUrl.EndsWith(".svg");
+    }
```

Comment:
Just a reminder that some URLs might not end with those extensions, but still be valid images. For example: https://www.dummyimage.com/600x400/000/fff

Is this what we want?